go/ql/src/experimental/CWE-134/DsnInjection.ql

```diff
@@ -13,14 +13,8 @@ import go
 import DataFlow::PathGraph
 import DsnInjectionCustomizations
 
-/** An argument passed via the command line taken as a source for the `DsnInjection` taint-flow configuration. */
-private class OsArgsSource extends Source {
-  OsArgsSource() {
-    exists(Variable c, SliceExpr s | c.hasQualifiedName("os", "Args") |
-      s.getBase() = c.getARead().asExpr() and this.asExpr() = s
-    )
-  }
-}
+/** An untrusted flow source taken as a source for the `DsnInjection` taint-flow configuration. */
+private class UntrustedFlowAsSource extends Source, UntrustedFlowSource { }
```

Comment:
```suggestion
private class UntrustedFlowAsSource extends Source instanceof UntrustedFlowSource { }
```

Makes clear this intends to enlarge `Source` with `UntrustedFlowSource`, not vice versa.